README.md

```diff
@@ -52,30 +40,89 @@ Ask someone to give you the token when in doubt.
 
 This needs to be done only once.
 
-## Installing / upgrading the package itsef
+## [Requirements](#requirements)
+
+#TODO: write requirements that the web delivery needs to satisfy in order for the module to work
+
+## [How to install](#how-to-install)
+> The module is designed to run in the browser. It can be used in node as well, but the primary target is  for  the browser.
+
+### [Installation via script tag](#install-via-script-tag)
+```html
+<script src="https://unpkg.com/@forward-distribution/bridget"></script>
+<script type="text/javascript">
+  // Initialize the module. Must be called before any other methods
+  bridget.init();
+</script>
+```
+Put the above script tags in between the <head> tags of your HTML page. It will attach a global `bridget` object. You can access it either via `bridget` or `window.bridget`.
+
+## [Alternative installation via NPM](#alternative-installation-via-npm)
+The module is available on NPM & as a github package which can be used if you’re using a front-end packager like Browserify or Webpack:
 
 Just run
 
 ```
 npm install -g @forward-distribution/bridget
 ```
 
-# Development
+You can then require the lib like a standard NPM module
+
+## [How to use](#how-to-use)
+### Reacting to user actions.
+Actions are something that a user performed on your website such as “Clicked on a teaser”, "Navigated to a page/department", or “Attempted to share”.
+
+In the context of your webpage, the browser itself or your custom JavaScript code handles these user events, and reacts to the accordingly. 
 
-Requirements:
+In the mobile context, however, these actions need to be propagated and offloaded from the browser (webview) and onto the app, to be handled natively. This is where you, as an integrator, will need to call bridget with the appropriate action (see [api reference](#api)).
+
+As an example, handling a click event on the share action of an article.
+
+```js
+	onClickEvent = e => {
+		e.preventDefault()
+		e.stopPropagation()
+		const { title, slug, description } = e.currentTarget.dataset
+		if(window.ReactNativeWebView)
```

Comment:
This type of code is also against the principles that were discussed (check Miro). We don't want the integrators to produce code that would require them to write "specifics" for webview.